pkg/operator/starter.go

```diff
@@ -85,6 +87,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		kubeClient,
 		kubeInformersForNamespaces.InformersFor(defaultNamespace),
 		nil,
+		csidrivercontrollerservicecontroller.WithSecretHashAnnotationHook(defaultNamespace, secretName, secretInformer),
```

Comment:
I think you should add `.WithExtraInformers(secretInformer.Informer())` below, as in AWS: https://github.com/openshift/aws-ebs-csi-driver-operator/blob/b05ef5e8dc1a4034e0d5a13f946fba2b43aa840e/pkg/operator/starter.go#L109-L109

This will postpone the sync loop until secret informer is fully loaded from API server.